waspc/e2e-test/test-outputs/waspBuild-golden/waspBuild/.wasp/build/server/src/server.js

```diff
@@ -7,6 +7,7 @@ import config from './config.js'
 
 
 const startServer = async () => {
+
```

Comment:
@Martinsos @shayneczyzewski Should we fix our templates and prevent them from adding empty lines when there's nothing to add?

Reply:
I think it may be helpful in future now that we have golden tests (to help keep them less noisy), but probably only worth a minor Issue so we don't forget and then just put it into Backlog. If you all agree I can add an Issue to track at least.

Reply:
Yes, that would be nice. It could be a simple good first issue.

Reply:
Actually, upon looking more closely here, I don't think it is an issue. The actual code diff looks like:
<img width="357" alt="Screen Shot 2022-05-10 at 2 17 24 PM" src="https://user-images.githubusercontent.com/523636/167695688-64fd951d-5f55-4390-87c1-0b59b71725e0.png">

So this just reflects the space that I added after the conditional. We should be good so I won't create an issue 👍🏻